### DISCUSSION TITLE
Line Level Outputs

### DISCUSSION THREAD

**Discussion by jberry** [micro-nova/AmpliPi#64] [Ideas] [answered]
How hard would it be to design a something to get line-level outputs for a zone?

I see two ribbon cables going into the amp, so I'm guessing right there would already be a line-level tie-in.

Would it be feasible to design an add-on module that'd basically sit between the cable and that jack, and provide access to the line-level signal, instead of using a speaker-to-line-level converter after the fact?

My use case is when I have an external amplifier for a zone or two, due to either the speaker wattage needed or the number of speakers.

**Comment by gorski123** [comment, marked as answer]
Hey jberry,
So funny you should ask, I am actually working on that right now. Line level outputs is something others have been asking for as well. And you are exactly correct, the top ribbon cable going to the amp board has the line level audio signals for the zones. I am currently working on modifying the amp board and adding a new circuit board (the preout board) that would provide a set of RCA line level outputs for each zone. Here is a picture of the new amp board, I added another connector (audio out) that provides access to the line level signals coming in on the 'audio in' connector.

![image](https://user-images.githubusercontent.com/20228663/108092117-6176d280-704a-11eb-8b9a-52a5097db847.png)

The new board will sit directly about the amp board and provide buffered line level outputs. You will be able to use either the amplified, line level, or both outputs for each zone. Here is a picture of the new preout board:

![image](https://user-images.githubusercontent.com/20228663/108092806-10b3a980-704b-11eb-9db6-9220e1021f72.png)

I will also add a copy of the preout board's schematic to the github repo if you want to check it out!

**Comment by jberry** [comment]
That's great, exactly what I was imagining. 

Do you think this will be part of the default device, or an add-on that'll need to be purchased? And if an add-on, will the holes at least be predrilled/plugged for easy plug-and-play?

**Reply by mabowers** [reply]
Hey @jberry, we added preamp outputs as a stretch goal for the Kickstarter, and we met it! So they will be part of the default device 🙂

The line level preamp outputs will exist as a pair RCA jacks directly above each of the 6 amplified zone outputs on the right side of the back panel. (After we finish up the case revisions, we will post a rendering of what the new rear panel looks like.)
